models/comments.models.js

```diff
@@ -17,3 +17,14 @@ exports.insertCommentsByArticle = (article_id, author, body) => {
     return result.rows[0];
   });
 };
+
+exports.removeCommentById = (comment_id) => {
+  text = "DELETE FROM comments WHERE comment_id = $1 RETURNING*"
+  params = [comment_id]
```

Comment:
I know that these variables work, but without const or let, they are global scoped and we don't want that! Please put ```const``` before them!